src/main/java/org/broadinstitute/hellbender/tools/spark/sv/discovery/inference/BreakpointComplications.java

```diff
@@ -1072,8 +1076,16 @@ public int hashCode() {
             result = 31 * result + (dupSeqRepeatUnitRefSpan != null ? dupSeqRepeatUnitRefSpan.hashCode() : 0);
             result = 31 * result + dupSeqRepeatNumOnRef;
             result = 31 * result + dupSeqRepeatNumOnCtg;
-            result = 31 * result + (dupSeqStrandOnRef != null ? dupSeqStrandOnRef.hashCode() : 0);
-            result = 31 * result + (dupSeqStrandOnCtg != null ? dupSeqStrandOnCtg.hashCode() : 0);
+            if (dupSeqStrandOnRef != null) {
+                for (final Strand strand : dupSeqStrandOnRef) {
+                    result = 31 * result + Objects.hashCode( strand.ordinal() );
+                }
+            }
+            if (dupSeqStrandOnCtg != null) {
+                for (final Strand strand : dupSeqStrandOnCtg) {
+                    result = 31 * result + Objects.hashCode( strand.ordinal() );
+                }
+            }
```

Comment:
This is the actual fix, I believe.  Chris and I missed the list of enums.  (A List's hashCode combines the hashCode's of its elements, and that's no good on Spark.)